src/solvers/prop/decision_procedure.h

```diff
@@ -34,8 +34,14 @@ class decision_proceduret
   void set_to_false(const exprt &expr);
 
   /// Convert a Boolean expression and return the corresponding literal
+  /// This will go away, use handle(expr) instead
```

Comment:
Mark as deprecated? The sooner we mark it as deprecated, the earlier we can remove it ;-)